fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/utils/FedoraTypesUtils.java

```diff
@@ -107,12 +107,16 @@
     });
 
     /**
+     * Check if a property is a reference property.
```

Comment:
I need to squash these commits anyways. I will go ahead and make the two minor updates mentioned in the comment above.